### ISSUE TITLE
segfault using session module

### ISSUE THREAD

**Issue by koleo** [aerospike/aerospike-client-php#124]
Like many other it seems, I am experiencing segfaults with Aerospike PHP client 3.4.11, using Aerospike Session module on a Symfony 2 project (PHP 5.5).

```
kernel: [41908.389656] php5-fpm[1523]: segfault at 0 ip 00007f350cd4cf2f sp 00007ffce4f0a108 error 4 in libc-2.13.so[7f350cc30000+184000]
```

After fixing a first segfault in `aerospike_helper_object_from_alias_hash` (conf->hosts is empty), a new segfault occurs further:

```
kernel: [42461.533277] php5-fpm[15147]: segfault at 0 ip 00007f45716bcceb sp 00007ffda6db2f00 error 4 in aerospike.so[7f4571643000+11b000]
```

I have not yet identified the exact problem, but it seems that Aerospike is loosing its config (connection string) for some reason.

However, I notice that segfault does not happen when I set `session.auto_start=true` in php.ini.

Still looking to fix this issue as soon as possible.

**Comment by koleo** [comment]
I fixed session module segfault, see PR #125

Thanks to @aterrien for his help on debugging this issue!

**Comment by jboone100** [comment]
Thanks much! Rolled into the 3.4.12 release.
